docs/promotions/free-trial-budget.rst

```diff
@@ -0,0 +1,57 @@
+.. _free-trial-budget:
+
+=================
+Free trial budget
+=================
+
+This section explains our Free trial budget mechanism. This feature gives the
```

Comment:
I would perhaps avoid mentioning this - we do give the credits to any new org the user creates, but we don't want them to be creating 10s of orgs to exploit this. Maybe something like "This features gives new users $200 of free credits, to be used with any cluster configuration for one month."?